src/components/Search/Search.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+
+
+class Search  extends Component  {
+    constructor() {
+        super();
+        this.state = {
+            foundPokemon:''
+        }
+    }
+
+    handleChange = (e) => {
+        this.setState({[e.target.name]: e.target.value})
+    }
+
+    handleClick = (e) => {
+        e.preventDefault();
+        const foundPokemon = {
+            id:Date.now(),
+            ...this.state
+        }
+        console.log(foundPokemon)
+        // foundPokemon will be = data.name 
+    }
+
+    render() {
+        return (
```

Comment:
Functional & ready for the next logic steps & styling.

Reply:
👍